rest/blip_client_test.go

```diff
@@ -661,6 +812,8 @@ func (btc *BlipTesterClient) PushRevWithHistory(docID, parentRev string, body []
 	proposeChangesRequest := blip.NewRequest()
 	proposeChangesRequest.SetProfile(db.MessageProposeChanges)
 	proposeChangesRequest.SetBody([]byte(fmt.Sprintf(`[["%s","%s","%s"]]`, docID, newRevID, parentRev)))
+	btc.addCollectionProperty(proposeChangesRequest)
```

Comment:
Does it make sense to do this inside `sendMsg` (with a special case for not doing it for `getCollections`)?

Having to specify this in every method feels error-prone or easy to miss it for new message types.

Reply:
I've had a go at this and couldn't really figure out a neat way of doing it. Besides requiring a user to pass in a collection to `sendMsg`. Do you think thats preferable?